src/components/captions-list/captions-list.tsx

```diff
@@ -0,0 +1,75 @@
+import {DownloadPluginManager} from '../../download-plugin-manager';
```

Comment:
are the lists similar enough to be merged into one component or are they too different ?

Reply:
@SivanA-Kaltura you mean the sources list and the captions list?

Reply:
yes